### ISSUE TITLE
Set up

### ISSUE THREAD

**Issue by Jordan-Pierce** [Jordan-Pierce/Marine-Debris#1]
@rosswin @abatt34 @kirakaplann

I'm going to start populating this repo with some things including:

- (general structure)
- a requirements.txt (please add your pip installs here)
- an install.py script
- a src folder, containing some reusable code from another Zooniverse project to be able to upload data to a Zooniverse project

I think it would be useful to communicate here in the future.

**Comment by Jordan-Pierce** [comment]
Okay, I added some initial set up stuff. I'm still in the process of trying to download a month's data from the Google drive.

**Comment by Jordan-Pierce** [comment]
@rosswin @abatt34 @kirakaplann

Made progress on downloading the data from the GDrive using the API. Unfortunately this is something that can really only be done from NOAA? So, I would say wrt to playing around with the code, it's best to manually download a month of data to test. Recommend using `April 2024`, as that's what Kira is using.

Will share updates.

**Comment by abatt34** [comment]
@Jordan-Pierce @rosswin @kirakaplann Played around w/ getting a raster dataset set up but am getting same errors that I know Kira is. seems like our JPEGs need to be georeferenced in order to use the "GeoDataset" type. I know I've done this within GIS platforms, but not in python. Thoughts? would we use GDAL? Anyone done this before? is this necessary?

**Comment by Jordan-Pierce** [comment]
Will check it out tomorrow. The JPEGs are georeferenced, but it could be stored in the header in a way that torchgeo isn't expecting

**Comment by rosswin** [comment]
Hey all,I haven’t taken a look yet, but usually these issues are due to missing georeference files. JPEGs don’t have a way to store internal georeference information (only the TIFF format can do that). Each image should have one, but possibly more, sidecar files such as .JGW or .JPG.AUX.XML. So make sure the folder that holds your JPEGs also has the matching sidecar files. If that doesn’t solve it then attach one of the troublesome images to this email and we should be able to solve that quickly. -RossOn Jul 7, 2024, at 9:41 PM, Jordan Pierce ***@***.***> wrote:﻿
Will check it out tomorrow. The JPEGs are georeferenced, but it could be stored in the header in a way that torchgeo isn't expecting

**Comment by Jordan-Pierce** [comment]
@rosswin 

Yep you're right. The files are JPEGs but do not have side-car files. I saw Lat / Lon in the EXIF, but they're just geotagged, not georeferenced. 

Below I have the annotations imported and converted as points (blue), and the images imported using the `GeoTaggedPhotosToPoints` tool. 

![Deer_Island_Test](https://github.com/Jordan-Pierce/Marine-Debris/assets/115024024/1922ba10-b50a-40b2-9bf1-52f514c01cee)

So, we need them to provide us with the side-car files for each JPEG image, or, more difficult, provide us with a GeoTIFF version of each image?

**Comment by abatt34** [comment]
seems like that's probably the best bet. Should we write an email? do you think they 100% have those files or should I research ways to do it ourselves?

**Comment by rosswin** [comment]
Without those sidecars we can’t get to true georeferencing. No way around it… gotta ask for those files. Once we have the sidecars we could convert the images to Tifs and embed the sidecar files into the image headers. Then we don’t have to worry about this. On Jul 8, 2024, at 2:51 PM, Allie Battista ***@***.***> wrote:﻿
seems like that's probably the best bet. Should we write an email? do you think they 100% have those files or should I research ways to do it ourselves?

**Comment by Jordan-Pierce** [comment]
Added tiling code
